lib/app/features/feed/stories/views/pages/story_record_page.dart

```diff
@@ -35,12 +38,22 @@ class StoryRecordPage extends HookConsumerWidget {
 
     final isCameraReady = cameraState is CameraReady;
 
-    ref.listen<StoryCameraState>(storyCameraControllerProvider, (_, next) {
+    ref.listen<StoryCameraState>(storyCameraControllerProvider, (_, next) async {
       if (next is StoryCameraSaved && context.mounted) {
-        StoryPreviewRoute(
-          path: next.file.path,
-          mimeType: next.file.mimeType,
-        ).push<void>(context);
+        var filePath = await ref.read(assetFilePathProvider(next.file.path).future);
+
+        if (filePath == null) return;
```

Comment:
Maybe show some error to the user?

Reply:
for now there is no such a product request. Also keep in mind, PO ll check this package it is not considired we ll use 100%.